### PR TITLE
RMMIS-5122-edit-entry-list

### PR DESCRIPTION
**Pull request by kjschulz**
RMMIS-5122: Add ability to edit entries in EntryList component

### CODE REVIEW

**Comment by kjschulz** [comment]
Alt text

File: src/EntryList.jsx [hunk 1]
```diff
@@ -1,157 +1,171 @@
...
+    this.state = {
       entries: [],
       entry: {},
-      showForm: false
+      showForm: false,
+      formConfig: {},
+      isEdit: false
     };
-  },
+    this.showEmptyText = this.showEmptyText.bind(this);
```
Comment (mallenscdhhs) [inline src/EntryList.jsx]: in lieu of having to `bind` all of these functions, why not make them `static` methods instead?

**Comment by mallenscdhhs** [comment]
@kjschulz it looks like you merged in your `FileList` code into this PR. That code hasn't been approved yet.

**Comment by kjschulz** [comment]
@michaelleeallen, awesome!  I'm a pro... :facepunch: Declining, will fix and reopen :crying_cat_face:

**Comment by mallenscdhhs** [comment]
:facepunch: :family: :cake:

**Pull request closed without merging**